python/tests/scripts/a3_twoside.metta

```diff
@@ -16,11 +22,16 @@
     ($x might be $P))
   ((Sam might be Frog)))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Composite queries
+; The comma is used to form multiple queries with shared
```

Comment:
Maybe, instead of just "multiple" something more specific could be used, e.g. "conjuncted queries" or "conjunction of queries"?